cluster/src/test/resources/scripts/cassandra_script1

```diff
@@ -0,0 +1,7 @@
+CREATE KEYSPACE test WITH replication = {'class':'SimpleStrategy', 'replication_factor' : 3};
```

Comment:
Why not use the same script (and table) for the cases i.e. snappyJobTest() and externalTableCreateTest()?